app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: 'from@example.com'
+  default from: 'from_email'
```

Comment:
this should be something like `Rails.application.secrets.from_email` instead. Or remove the entire line.